core/src/main/java/de/javakaffee/web/msm/StorageClientFactory.java

```diff
@@ -43,6 +43,15 @@ MemcachedClient createCouchbaseClient(MemcachedNodesManager memcachedNodesManage
                 long maxReconnectDelay, Statistics statistics );
     }
 
+    static interface MemcachedNodeURLBasedConnectionFactory {
+        ConnectionFactory createBinaryConnectionFactory(long operationTimeout,
+                                                        long maxReconnectDelay, boolean isClientDynamicMode);
```

Comment:
The `isClientDynamicMode` should not be part of this API, because its specific to the ElastiCache specific implementation.